src/main/java/net/sf/jabref/logic/formatter/bibtexfields/NormalizePagesFormatter.java

```diff
@@ -58,7 +58,7 @@ public String format(String value) {
         }
 
         // remove unwanted literals incl. whitespace
-        String cleanValue = value.replaceAll(REJECT_LITERALS, "");
+        String cleanValue = value.replaceAll("\u2013|\u2014", "-").replaceAll(REJECT_LITERALS, "");
```

Comment:
Could you create a constant for the regex to give it a name, something like REJECT_LONG_DASHES?